src/containers/SearchPage/components/form/SearchConceptForm.tsx

```diff
@@ -101,7 +102,22 @@ const SearchConceptForm = ({ search: doSearch, searchObject: search, subjects }:
     };
   };
 
+  const conceptTypes = useMemo(
+    () => [
+      { id: 'concept', name: t('searchForm.conceptType.concept') },
+      { id: 'gloss', name: t('searchForm.conceptType.gloss') },
+    ],
+    [t],
+  );
+
   const selectors: SearchFormSelector[] = [
+    {
+      parameterName: 'concept-type',
+      value: getTagName(search['concept-type'], conceptTypes),
+      options: conceptTypes,
+      formElementType: 'dropdown',
+      width: 50,
```

Comment:
Gir ingen meining at en dropdown med valga "Forklaring" og "Glose" skal være 50%. Bedre å ha 25 og dermed få to linjer med filtre.
```suggestion
      width: 25,
```